core/src/test/java/org/apache/calcite/test/TypeCoercionTest.java

```diff
@@ -894,6 +894,13 @@ public MockCatalogReader init() {
       t2.addColumn("t2_binary", binaryType);
       t2.addColumn("t2_boolean", booleanType);
       registerTable(t2);
+
+      final MockTable t3 =
+          MockTable.create(this, tSchema, "T3", false, 7.0, null);
+      t3.addColumn("t3_varchar20", varchar20Type);
+      t3.addColumn("t3_int", intType);
```

Comment:
Why add a new table here ? 't1' and 't2' have all the data types you needed.

Reply:
I'm worried that too **many columns** will affect everyone's reading, so I only use **a few columns.** If you think it is more appropriate to use the existing table, I can change it later.

Reply:
Yeah, i'm sliding to reuse the original tables.

Reply:
I deleted `t3` and used `t1`.